core-services/prow/02_config/quay/quay-bridge-operator/_pluginconfig.yaml

```diff
@@ -0,0 +1,50 @@
+approve:
+- commandHelpLink: https://go.k8s.io/bot-commands
+  lgtm_acts_as_approve: true
+  repos:
+  - quay/quay-bridge-operator
+  require_self_approval: false
+external_plugins:
+  quay/quay-bridge-operator:
+  - endpoint: http://refresh
+    events:
+    - issue_comment
+    name: refresh
+  - endpoint: http://cherrypick
+    events:
+    - issue_comment
+    - pull_request
+    name: cherrypick
+  - endpoint: http://needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+    name: needs-rebase
+plugins:
+  quay/quay-bridge-operator:
+    plugins:
+    - assign
```

Comment:
You are configuring an `approve` plugin on L1 but do not enable it

Reply:
updated